service/frontend/api/handler.go

```diff
@@ -581,7 +582,24 @@ func (wh *WorkflowHandler) PollForActivityTask(
 			return nil, err
 		}
 	}
-	return resp, nil
+	return &types.PollForActivityTaskResponse{
```

Comment:
we may miss to update here when new fields are added. better to define From/To funcs and have unit tests.